packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -79,7 +79,7 @@ const replaceJSON = (path, replacements) =>
 const slugifyName = (name) => {
     return slugify(name, {
         lower: true,
-        strict: true
```

Comment:
Do we want trailing commas? I think we set it to be off

Reply:
fixed